qa/integration/services/dockerized/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stretch
+##
+# Define a base image for all service images.
+##
+
+ENV _JAVA_OPTIONS "-Djava.net.preferIPv4Stack=true"
+ENV TERM=linux
+
+RUN apt-get update && apt-get install -y curl openjdk-8-jre-headless netcat
+
+RUN adduser --disabled-password --gecos '' tester
```

Comment:
@wainersm do we need a user other than root here, maybe it's easier to just run ES with `-Des.insecure.allow.root=true` (I don't have a strong argument here (yet) but we're seeing some issues with testing various networking things, that will probably require running shell scripts from Ruby through the Docker api, would be nice if we had `root` as a user throughout here if possible) ?